framework/test/unit/modules/pos/module.spec.ts

```diff
@@ -127,6 +127,36 @@ describe('PoS module', () => {
 
 			expect(pos['_moduleConfig'].maxLengthName).toBe(50);
 		});
+
+		it('should throw when configured values for failSafeInactiveWindow or punishmentWindow are outside of their allowed ranges', async () => {
```

Comment:
Do you believe adding the error message thrown by validator would increase readability and helps against scenarios where test passes even though the error thrown is different than the scenario expected?

Reply:
Latest commit doesn't provide much info on the field that had the error. Interestingly, the error reported by validation logic is also not concrete, somehow; the field name is missing from the error message; like;
```
    Expected substring: "failSafeInactiveWindow"
    Received message:   "Lisk validator found 1 error[s]:
    must be >= 144000"
```

Perhaps, it can be investigated; why the property name is missing, but to the best of my knowledge would be way off the PR's scope, in that case, the addition of error message check should be reverted.